lighthouse-core/computed/metrics/layout-stability.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const makeComputedArtifact = require('../computed-artifact.js');
+const MetricArtifact = require('./metric');
+const LHError = require('../../lib/lh-error');
+
+class LayoutStability extends MetricArtifact {
```

Comment:
we probably don't need to bother with extending metricartifact since there's no difference between lantern and observed. it's only useful when we need to utilize the graph and account for both modes.

this should also help eliminate the `.timing` issue

Reply:
good call. didnt realize lantern-speed-index and a few others were like this.

score.